docs/database_schema_diagrams/write_graphs.pl

```diff
@@ -281,7 +282,7 @@ sub write_dot_files
         if (exists $diagram_columns{$table}) {
             my @column_names = @{ $diagram_columns{$table} };
             foreach my $col (@column_names) {
-                if (any { $_ eq $col } @{ $created_primary_keys{$table} }) {
+                if (contains_string([ $created_primary_keys{$table} ], $col)) {
```

Comment:
`$created_primary_keys{$table}` is a scalar
```suggestion
                if (contains_string($created_primary_keys{$table}, $col)) {
```